<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="serverView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="cmd"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="send">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStart_Server"/>
    <addaction name="actionStop_Server"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionStart_Server">
   <property name="text">
    <string>Start Server</string>
   </property>
  </action>
  <action name="actionStop_Server">
   <property name="text">
    <string>Stop Server</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
